expo/expo
# [expo-constants] Modularized  expo-constants without further app setup

# Why

Try to achieve the goal of `expo install expo-constants` without any further setup.

# How

For iOS, use `script_phase` to execute the script.
For Android, move gradle plugin into expo-constants android library.

# Test Plan

1. Build passed
2. Verify Constants.Manifest test case from bare-expo iOS/Android 

# Checklist
